json_container/inc/leatherman/json_container/json_container.hpp

```diff
@@ -20,6 +20,9 @@ namespace rapidjson {
 }  // namespace rapidjson
 
 namespace leatherman { namespace json_container {
+    // Constants
+    const size_t DEFAULT_LEFT_PADDING { 4 };
```

Comment:
This must be `static const` or some compilers will warn of unused variables (since it's in a namespace; in a class that would be a different matter).

Reply:
You could also use constexpr, which doesn't need to be static.